partiql-planner/src/main/resources/partiql_plan_internal.ion

```diff
@@ -278,10 +278,6 @@ rel::{
     limit::{
       input:  rel,
       limit:  rex,
-    },
-
-    offset::{
-      input:  rel,
       offset: rex,
     },
```

Comment:
LIMIT NULL is not the same as LIMIT 0.

I had originally modeled LIMIT..OFFSET as the FETCH operator, because FETCH is defined in the SQL specification whereas LIMIT and OFFSET are not actually defined the SQL spec (92 or 99). 

However, PartiQL allows for OFFSET without LIMIT hence why I split them into their own operators. I don't think we should consolidate, but if that is agreed to be better then it should at least be called FETCH